backend/entityservice/tests/util.py

```diff
@@ -263,7 +263,7 @@ def wait_approx_run_time(size, assumed_rate=1_000_000):
     """
     number_comparisons = sum(
         size0 * size1 for size0, size1 in itertools.combinations(size, 2))
-    time.sleep(5)
+    time.sleep(1)
```

Comment:
Relates to issue #457

Reply:
As I wasn't sure why it has been introduced, I reduced it to 1 second and opened an issue. I'll check what happens without this wait (f7601d179c07156d379af9a2dcd2081e3b8c2448), if the tests fail, I'll revert it, otherwise keep it.